engine/liveupdate/src/liveupdate.cpp

```diff
@@ -560,8 +524,17 @@ printf("MAWE: StoreManifest: %p  algorithm: %d\n", manifest, (int)algorithm);
             result = LULoadManifest_Regular(archive_name, app_path, app_support_path, base_manifest, out);
             if (dmResourceArchive::RESULT_OK != result)
             {
-                LUCleanup_Regular(archive_name, app_path, app_support_path);
-                g_LiveUpdate.m_ArchiveType = -1;
+                if (dmResourceArchive::RESULT_NOT_FOUND != result)
+                {
+                    LUCleanup_Regular(archive_name, app_path, app_support_path);
+                    g_LiveUpdate.m_ArchiveType = -1;
+                }
+                else
+                {
+                    // We might download a manifest later, so we shouldn't clean the currently downloaded archive.
+                    // This is the behavior from 1.2.174 and before.
+                    result = dmResourceArchive::RESULT_OK;
```

Comment:
This was a use case I hadn't considered, that it is possible to download new resources, but not a new manifest.